Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -545,6 +600,41 @@ public class Fuzzer {
         }
     }
 
+    func processDifferential(_ program: Program, withStderr stderr: String, withStdout stdout: String, origin: ProgramOrigin) {
+        func processCommon(_ program: Program) {
+            let hasDiffInfo = program.comments.at(.footer)?.contains("DIFFERENTIAL INFO") ?? false
+            if !hasDiffInfo {
+                program.comments.add("DIFFERENTIAL INFO\n==========\n", at: .footer)
+
+                program.comments.add("STDERR:\n" + stderr, at: .footer)
+                program.comments.add("STDOUT:\n" + stdout, at: .footer)
+                program.comments.add("ARGS: \(runner.processArguments.joined(separator: " "))\n", at: .footer)
+                program.comments.add("REFERENCE ARGS: \(referenceRunner!.processArguments.joined(separator: " "))\n", at: .footer)
+            }
+            assert(program.comments.at(.footer)?.contains("DIFFERENTIAL INFO") ?? false)
+
+            // Check for uniqueness only after minimization
+            let execution = execute(program, withTimeout: self.config.timeout * 2)
+            if case .differential = execution.outcome {
+                // TODO don't use crash eval
```

Comment:
I agree. I'm not sure there's any value in trying to determine if a differential is unique (but maybe there is...?), so I'm also fine with just treating all differential programs as unique.

Reply:
I'll update the PR s.t. all differentials are treated as unique; we'd only indicate whether they're flaky or deterministic.